test/skate_web/controllers/page_controller_test.exs

```diff
@@ -52,6 +52,29 @@ defmodule SkateWeb.PageControllerTest do
       assert html =~ "selected_route_ids&quot;:[&quot;1&quot;]"
     end
 
+    @tag :authenticated
+    test "includes user test groups in HTML", %{conn: conn, user: user} do
+      user_struct = user |> Skate.Settings.User.get()
+
+      Skate.Settings.TestGroup.update(%{
+        Skate.Settings.TestGroup.create("html-test-group")
+        | users: [user_struct]
+      })
```

Comment:
The test succeeds with multple test groups defined, but it's somewhat verbose. I'm wondering if anyone else thinks we should be testing with multiple groups in this test or create another one that does that?

Reply:
I'm not overly concerned but testing with 0, 1, and N would certainly be nice. I don't think ExUnit really has built-in test parametrization tools, but your test can always wrap some assertions in a `for` block.